### ISSUE TITLE
Option to start empty

### ISSUE THREAD

**Issue by akaleeroy** [gskinner/regexr#249]
I signed up (with GitHub) hoping that I could make it remember an empty state as default. It would save 2 seconds if users didn't have to Select all in the Text field, delete, then go up to the pattern and delete that  too. 

Please add this option.

**Comment by gskinner** [comment]
I can totally see the value in this suggestion. Right now we don't have any mechanism for user preferences, but I'll keep this open for future consideration.

**Comment by JMichaelTX** [comment]
> if users didn't have to Select all in the Text field, delete, then go up to the pattern and delete that too.
> Please add this option.

I totally agree.  If you can't give us the option now, then just change the default to empty.  There's plenty of help on the side if the user needs it.  That's how Regex101.com works.

**Comment by gskinner** [comment]
I think there's a lot of value in having the initial text / pattern for people who are visiting the site for the first time.

A couple of changes that I think could work as a compromise (at least until we have user prefs) would be:

1. Make "New" patterns blank.
2. Have it start blank if you're signed in.

Would that work for now?

**Comment by akaleeroy** [comment]
Yep, either (or both) of those would solve the issue.

**Comment by RegexPro** [comment]
Hi GSkinner, thanks for addressing the above request!  However, when I log in or press new this is what I get instead of the expected blank fields:
**Expression:** .
**Text:** Text
